app/services/analytics_events.rb

```diff
@@ -465,6 +465,134 @@ def idv_doc_auth_exception_visited(step_name:, remaining_attempts:, **extra)
     )
   end
 
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [Integer] attempts
+  # @param [Integer] remaining_attempts
+  # @param [String] user_id
+  # @param [String] flow_path
+  # The document capture  image uploaded was locally validated during the IDV process
```

Comment:
supernit... extra space 
```suggestion
  # The document capture image uploaded was locally validated during the IDV process
```